From d21f60d4a8bb03e0309fe3907b104f1878059f90 Mon Sep 17 00:00:00 2001
From: Naveen M K <naveen521kk@gmail.com>
Date: Sat, 2 Nov 2024 11:30:54 +0530
Subject: [PATCH 119/N] socketmodule: fix captilization of headers

`Rpc` should be `rpc`. It error's out in
cross-compilation.
---
 Modules/socketmodule.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Modules/socketmodule.c b/Modules/socketmodule.c
index 1f7f297..6fa440a 100644
--- a/Modules/socketmodule.c
+++ b/Modules/socketmodule.c
@@ -279,7 +279,7 @@ shutdown(how) -- shut down traffic in one or both directions\n\
 # endif
 
 /* Helpers needed for AF_HYPERV */
-# include <Rpc.h>
+# include <rpc.h>
 
 /* Macros based on the IPPROTO enum, see: https://bugs.python.org/issue29515 */
 #ifdef _MSC_VER
